src/styles/blocks/button.scss

```diff
@@ -0,0 +1,40 @@
+.button {
+  position: relative;
+  z-index: 1;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  box-sizing: border-box;
+  height: 56px;
+  width: 100%;
+  font-weight: 700;
+  text-decoration: none;
+  color: #fff;
+  background-color: $colorBlack;
+  border-radius: 40px;
+  cursor: pointer;
+
+  transition: all 0.3s;
+
+  &--primary {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 27px;
+    color: $colorBlack;
+    background-color: transparent;
+    border: 3px solid $colorBlack;
+  }
+
+  &:hover {
+    color: $colorBlack;
+    background-color: transparent;
+    border: 3px solid $colorBlack;
+  }
+
+  &:active {
+    background-color: #111;
+    color: white;
```

Comment:
Use SCSS variables for colors. Replace `background-color: #111` with a variable like `$colorDark`. 🌈